pkg/ui/react-app/src/thanos/pages/blocks/BlockDetails.tsx

```diff
@@ -53,7 +54,7 @@ export const BlockDetails: FC<BlockDetailsProps> = ({ block, selectBlock }) => {
           </div>
           <hr />
           <div data-testid="labels">
-            <b>Labels:</b>
+            <b>Labs:</b>
```

Comment:
Seems like you changed it by mistake. It should be `Labels:`, right?

Reply:
Ah  yes this was a mistake, Fixed.